### ISSUE TITLE
Reusable die of fate

### ISSUE THREAD

**Issue by Durkel** [tgstation/tgstation#19796]
If you somehow manage to get your hands on a die of fate you can use it, turn yourself into a monkey, re humanize yourself and use it again. Continue doing this until you become a wizard or a pile of ash.

**Comment by KorPhaeron** [comment]
Where did the die of fate come from?

**Comment by Incoming5643** [comment]
It was the prize in the wizard academy away mission

**Comment by KorPhaeron** [comment]
No I mean this particular one. Need to know if it was the reusable version or not, and if so why there is a reusable version still accessible.

**Comment by Durkel** [comment]
Alright, after finally updating my server and wasting 20 minutes to compile I spawned the greed slot machine to test this out. It worked properly as a one use as I can tell, so the die that was being used must have been something from a funmin.

So yeah, looks like it was actually a reusable die of fate spawned by someone rather than a bug. Either that or the miner I saw it with somehow managed to get his hands on a reusable version. I'll ask him next time if I ever see him again, honk.
